fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -10,7 +10,11 @@ def self.run(params)
         begin
           path = File.expand_path(params[:path])
           plist = Plist.parse_xml(path)
-          plist[params[:key]] = params[:value]
+          if params[:subkey]
+            plist[params[:key]][params[:subkey]] = params[:value]
```

Comment:
Ok, I've done another test: It crashes when `key` is not available _and_ `subkey` is provided. I'll go ahead and fix this and provide update here. Sorry.